cgo/lapack.go

```diff
@@ -542,6 +543,47 @@ func (impl Implementation) Dgetrs(trans blas.Transpose, n, nrhs int, a []float64
 	clapack.Dgetrs(trans, n, nrhs, a, lda, ipiv32, b, ldb)
 }
 
+// Dorgbr generates one of the matrices Q or P^T computed by Dgebrd
+// computed from the decomposition Dgebrd. See Dgebd2 for the description of
```

Comment:
I was referring to the `... computed by Dgebrd computed from the decomposition Dgebrd. ...` part.